tests/tools/test_ontoconvert.py

```diff
@@ -0,0 +1,14 @@
+"""Test the `ontoconvert` tool."""
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.parametrize("tool", ["ontoconvert"], indirect=True)
+def test_run(tool, tmpdir: Path) -> None:
+    """Check running `ontoconvert` works."""
+    test_file = (
+        Path(__file__).resolve().parent.parent / "testonto" / "models.ttl"
+    )
+
+    tool.main([str(test_file), str(tmpdir / "test.ttl")])
```

Comment:
As far as I can see this only checks that `ontoconvert` runs. Shouldn't the result also be checked?

Reply:
No, the issue here was that running `ontoconvert` was failing - so here I've created extremely simple tests to make sure it can actually run as intended. Then we can make extra tests to check the contents and logic as well. I think modularizing tests this way makes it easier to manage the testing suite, as each test focuses on a single (simple) feature that it's checking. Hopefully that will make it easier to pinpoint bugs in the future as well :)